src/tensors/abstracttensor.jl

```diff
@@ -23,15 +23,59 @@ i.e. a tensor map with only a non-trivial output space.
 const AbstractTensor{S<:IndexSpace,N} = AbstractTensorMap{S,N,0}
 
 # tensor characteristics
+#------------------------
 Base.eltype(::Union{T,Type{T}}) where {T<:AbstractTensorMap} = scalartype(T)
+
+"""
+    spacetype(::Union{T,Type{T}}) where {T<:AbstractTensorMap} -> Type{S<:IndexSpace}
+
+Return the type of the elementary space `S` of a tensor.
+"""
 spacetype(::Type{<:AbstractTensorMap{S}}) where {S<:IndexSpace} = S
+
+"""
+    sectortype(::Union{T,Type{T}}) where {T<:AbstractTensorMap} -> Type{I<:Sector}
+
+Return the type of sector `I` of a tensor.
+"""
 sectortype(::Type{<:AbstractTensorMap{S}}) where {S<:IndexSpace} = sectortype(S)
+
 function InnerProductStyle(::Type{<:AbstractTensorMap{S}}) where {S<:IndexSpace}
     return InnerProductStyle(S)
 end
+
+"""
+    field(::Union{T,Type{T}}) where {T<:AbstractTensorMap} -> Type{𝕂<:Field}
+
+Return the type of field `𝕂` of a tensor.
```

Comment:
I am definitely in favor of doing this. I think it is now only being used in checking if the deligne product is allowed, i.e. it will not allow you to take the deligne product of `CartesianSpace` and `ComplexSpace`, but I am not sure if this check is really all that useful.